### PR TITLE
test(prune): add unit tests for `ReceiptsLogPruneConfig`

### PR DESCRIPTION
**Pull request by tcoratger**

### CODE REVIEW

File: crates/prune/types/src/mode.rs [hunk 1]
```diff
@@ -37,6 +37,7 @@ impl PruneMode {
             Self::Full if segment.min_blocks(purpose) == 0 => Some((tip, *self)),
             Self::Distance(distance) if *distance > tip => None, // Nothing to prune yet
             Self::Distance(distance) if *distance >= segment.min_blocks(purpose) => {
+                println!("iciiiiiiiii: {}", tip - distance);
```
Comment (mattsse) [inline crates/prune/types/src/mode.rs:40]: leftover

**Review by mattsse — CHANGES_REQUESTED** [review]
lgtm, ty

pending rm println

File: crates/prune/types/src/mode.rs (hunk 1 above)
Comment (tcoratger) [reply crates/prune/types/src/mode.rs]: oups sorry removed

**Comment by tcoratger** [comment]
hey @mattsse nice bump here, anything missing?

**Review by mattsse — APPROVED** [review]

**Pull request merged**